library/alignment.h

```diff
@@ -37,6 +37,19 @@
 #define MBEDTLS_EFFICIENT_UNALIGNED_ACCESS
 #endif
 
+#if defined(__IAR_SYSTEMS_ICC__)
+/* IAR recommend this technique for accessing unaligned data in
+ * https://www.iar.com/knowledge/support/technical-notes/compiler/accessing-unaligned-data
```

Comment:
Hmm, I see them recommending `uint32_t __packed *`; the word `union` doesn't appear on that page

Reply:
They recommend it for a struct: https://www.iar.com/knowledge/support/technical-notes/compiler/accessing-unaligned-data#:~:text=UNALIGN_TRP%20bit%E2%80%9D.-,Structures,-Using%20%23pragma%20pack - we could use a struct if you prefer, but this seemed tidier.